test-framework/maven/src/main/java/io/quarkus/maven/it/verifier/RunningInvoker.java

```diff
@@ -119,6 +120,7 @@ public MavenProcessInvocationResult execute(List<String> goals, Map<String, Stri
             //just running GC
             request.setMavenOpts("-Xmx128m");
         }
+        MojoTestBase.passUserSettings(request);
```

Comment:
In conjunction with https://github.com/quarkusio/quarkus/pull/15530/files#r590560562, maybe that method should be located in some util class in the `verifier` package instead of the test base class?

Reply:
I'm also wondering whether `MavenProcessInvoker` should do the same.

Reply:
That's pretty much where this is used though. I liked the idea of wrapping the invoker. And the RunningInvoker is already overriding `execute`.

Reply:
I don't see MavenProcessInvoker being used directly though.